src/main/java/pl/umk/mat/git2befit/service/workout/factory/implementation/FBWTrainingPlan.java

```diff
@@ -21,6 +22,7 @@
     private List<Exercise> exercisesWithEquipment = new ArrayList<>();
     private TrainingForm trainingForm;
     private TrainingForm localTrainingForm;
+    private List<Training> trainingList;
```

Comment:
Na razie niech to zostanie, ale ja to przerobię trochę w następnym sprincie i podepnę to pod to, co ja zrobiłem